src/components/Resources/index.js

```diff
@@ -5,23 +5,44 @@ import PersonalMenu from '../PersonalMenu';
 import Search from '../Search';
 import Grid from '@material-ui/core/Grid';
 import { ResourceCard } from './ResourceCard';
+import * as helper from '../helper';
```

Comment:
Prefer specified imports because [tree shaking to minimise bundle size](https://material-ui.com/guides/minimizing-bundle-size/)

Reply:
Sounds good, thanks! Done in 73e9562